src/components/WebexAdaptiveCard/WebexAdaptiveCard.jsx

```diff
@@ -19,44 +20,70 @@ import AdaptiveCard from '../adaptive-cards/AdaptiveCard/AdaptiveCard';
  * @param {object} props  Data passed to the component
  * @param {string} props.activityID  ID of the activity corresponding to this card
  * @param {string} [props.className]  Custom CSS class to apply
+ * @param {string} [props.msgSubmitStarted]  Sending message
```

Comment:
Message to display while submitting user input
Message to display when submit finished with success
Message to display when submit failed